kubernetes/EKS/templates/eks_cluster.yaml

```diff
@@ -4,12 +4,20 @@ kind: ClusterConfig
 metadata:
   name: "TorchserveCluster"
   region: "us-west-2"
-  
-nodeGroups:
-  - name: ng-1
-    instanceType: g4dn.xlarge
-    desiredCapacity: 1
+
+managedNodeGroups:
+  - name: mg-1
+    # EKS Optimized AMIs - https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html
+    ami: ami-048af94822f84c9c1
+    instanceType: t3.xlarge
+    minSize: 1
+    desiredCapacity: 2
+    maxSize: 3
+    overrideBootstrapCommand: |
+      #!/bin/bash
+      /etc/eks/bootstrap.sh <cluster-name> --container-runtime containerd
```

Comment:
Is the user expected to replace `<cluster-name>` before using this template? If so, please add it in the docs.